docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/armis-api-source.md

```diff
@@ -0,0 +1,84 @@
+
+## Create an Armis Source
+
+When you create a Armis Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+To configure a Armis Source:
+1. In Sumo Logic, select **Manage Data > Collection > Collection**. 
+1. On the Collection page, click **Add Source** next to a Hosted Collector.
+1. Select **Armis**.<br/> <img src={useBaseUrl('img/send-data/armis-icon.png')} alt="armis-icon.png" width="150" />
+1. Enter a **Name** for the Source. The description is optional.<br/> <img src={useBaseUrl('img/send-data/armis-config-main.png')} alt="armis-config-main" width="450" />
+1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+1. (Optional) **Fields.** Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored (i.e., dropped).
+1. In **Instance URL**, enter the Armis hostname. For example, https://armis-instance.armis.com.
+ :::note
+ Armis Instance URL is the Armis hostname.
+ :::
+1. In **Secret Key**, enter your API secret key you have generated in [Setup and Configuration](#setup-and-configuration) section.
+1. In **Armis API selection**. Choose the data sources from which you want to ingest data. The integration provides the option to you to select either one or both of the data sources.
+   * If **Alert API** is selected, the integration will fetch alerts data.
+     * Permission `Alert>Read` must be provided to fetch alerts data.
+     * Data for alert will be fetched every 5 minutes.
+   * If **Device API** is selected, the integration will fetch device data.
+     * Permission `Device>Read` must be provided to fetch device data.
+     * Data for device will be fetched every 24 hours.
+:::note
+This step is mandatory, ensure you select one data source.
+:::
+1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allow list, deny list, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

Comment:
allow list, deny list are one word (_allowlist_, _denylist_)

```suggestion
1. (Optional) In **Processing Rules for Logs**, configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

Reply:
@kimsauce Thank you for the suggestions.